changelog/17071.txt

```diff
@@ -0,0 +1,2 @@
+```release-note:feature
+**UI OIDC Provider Config**: Adds configuration of Vault as an OIDC identity provider, and offer Vault’s various authentication methods and source of identity to any client applications.
```

Comment:
Should this be something like: `ui: xyz`? See the following recent changelog for ui changes.

```release-note:bug
ui: Fix OIDC callback to accept namespace flag in different formats
```

Reply:
For larger features I think the format is slightly different. Dm-ing you the internal docs